make_function_lists.R

```diff
@@ -0,0 +1,256 @@
+    group_by(package) %>% 
+    slice(1) %>% 
+    ungroup()
+  x %>% 
+    left_join(pkgdown_urls, by = "package") %>% 
+    mutate(
+      first_url = map_chr(all_urls, ~ .x[1]),
+      first_url = ifelse(is.na(first_url), 
+                         glue("https://cran.r-project.org/package={package}"), 
+                         first_url),
+      base_url = ifelse(is.na(pkgdown_url), 
+                        first_url, 
+                        pkgdown_url),
+      url = ifelse(!is.na(pkgdown_url), 
+                   glue("{pkgdown_url}/reference/{file_out}"),
+                   base_url),
+      topic = glue("<a href='{url}' target='_blank'><tt>{functions}</tt></a>")
+    ) %>% 
+    dplyr::select(title, functions, topic, package) %>% 
+    mutate(package = as.factor(package)) %>% 
+    filter(!grepl("deprecated", tolower(title))) %>% 
+    arrange(tolower(gsub("[[:punct:]]", "", title)))
+}
+
+# ------------------------------------------------------------------------------
+
+broom_pkgs <- revdepcheck::cran_revdeps("broom",    dependencies = c("Depends", "Imports"))
+generics_pkgs <- revdepcheck::cran_revdeps("generics", dependencies = "Imports")
+
+broom_pkgs <- sort(unique(c(broom_pkgs, generics_pkgs)))
+excl <- c("hydrorecipes", "healthcareai")
+broom_pkgs <- broom_pkgs[!(broom_pkgs %in% excl)]
+
+broom_functions <- 
+  map_dfr(
+    broom_pkgs, 
+    get_pkg_info, 
+    pattern = "(^tidy\\.)|(^glance\\.)|(^augment\\.)",
+    .progress = TRUE
+  ) %>% 
+  sort_out_urls() %>% 
+  select(-functions)
+
+save(
+  broom_functions, 
+  file = "content/find/broom/broom_functions.RData", 
+  compress = TRUE)
+
+# ------------------------------------------------------------------------------
+
+recipe_pkgs <- revdepcheck::cran_revdeps("recipes", dependencies = c("Depends", "Imports"))
+
+recipe_pkgs <- sort(unique(c(recipe_pkgs)))
+excl <- c("hydrorecipes", "healthcareai")
+recipe_pkgs <- recipe_pkgs[!(recipe_pkgs %in% excl)]
+
+
+recipe_functions <- 
+  map_dfr(
+    recipe_pkgs, 
+    get_pkg_info, 
+    pattern = "^step_",
+    .progress = TRUE
+  )  %>% 
+  sort_out_urls() %>% 
+  select(-functions)
+
+save(
+  recipe_functions, 
+  file = "content/find/recipes/recipe_functions.RData", 
+  compress = TRUE)
+
+# ------------------------------------------------------------------------------
+
+all_tm <- 
+  c("agua", "applicable", "baguette", "brulee", "broom", "butcher", 
+    "censored", "corrr", "dials", "discrim", "embed", "finetune", 
+    "hardhat", "infer", "modeldata", "modeldb", 
+    "modelenv", "multilevelmod", "parsnip", "plsmod", "poissonreg", 
+    "probably", "recipes", "rsample", "rules", "shinymodels", "spatialsample", 
+    "stacks", "textrecipes", "themis", "tidyclust", "tidymodels", 
+    "tidyposterior", "tidypredict", "tune", "usemodels", "workflows", 
+    "workflowsets", "yardstick")
+
+
+tidymodels_functions <- 
+  map_dfr(
+    all_tm, 
+    get_pkg_info, 
+    .progress = TRUE
+  ) %>% 
+  sort_out_urls() %>% 
```

Comment:
```suggestion
  sort_out_urls() %>% 
  filter(grepl("^\\.", functions)) %>%
```

This code should do the trick to remove functions marked as internal👍